### PR TITLE
More operators

### PR DESCRIPTION
**Pull request by IainNZ**
Continuing on #6

### CODE REVIEW

**Comment by mlubin** [comment]
WIP or ready to merge?

File: src/operators.jl [hunk 1]
```diff
@@ -15,48 +15,44 @@
...
 
 (-)(pos::Posynomial, num::Number) = pos - Monomial(num)
 (-)(pos::Posynomial, mon::Monomial) = pos + (-1)*mon
-(-)(mon::Monomial, pos::Posynomial) = mon + (-1)*pos
-function (*)(pos::Posynomial, m::Monomial)
-    return Posynomial([mon*m for mon in pos.mons])
-end
-(*)(mon::Monomial,pos::Posynomial) = pos*mon
-
+(*)(pos::Posynomial, m::Monomial) =m * pos
```
Comment (mlubin) [inline src/operators.jl]: Weird spacing here

**Comment by IainNZ** [comment]
Still got to push the posynomial-- commit then done

**Comment by IainNZ** [comment]
OK, done now

**Comment by mlubin** [comment]
:+1:

**Pull request merged**